docs/LoongArch-ELF-ABI-CN.adoc

```diff
@@ -647,6 +647,203 @@ v1.00
 |R_LARCH_GNU_VTENTRY
 |GNU C++ vtable 支持
 |
+
+4+|... 保留项
+
+|64
+|R_LARCH_B16
+|18 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 18 位有符号数溢出和4字节对齐检测功能
+
+|65
+|R_LARCH_B21
+|23 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [4 ... 0] = (S+A-PC) [22 ... 18],+`
+
+`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 23 位有符号数溢出和4字节对齐检测功能
+
+|66
+|R_LARCH_B26
+|28 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [9 ... 0] = (S+A-PC) [27 ... 18],+`
+
+`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 28 位有符号数溢出和4字节对齐检测功能
+
+|67
+|R_LARCH_ABS_HI20
+|32 位绝对地址的[12:31]位
+|`(*(uint32_t *) PC) [24 ... 5] = (S+A) [31 ... 12]+`
+
+|68
+|R_LARCH_ABS_LO12
+|32 位绝对地址的[0:11]位
+|`(*(uint32_t *) PC) [21 ... 10] = (S+A) [11 ... 0]+`
+
+|69
+|R_LARCH_ABS64_LO20
+|64 位绝对地址[32:51]位
+|`(*(uint32_t *) PC) [24 ... 5] = (S+A) [51 ... 32]+`
+
+|70
+|R_LARCH_ABS64_HI12
+|64 位绝对地址[52:63]位
+|`(*(uint32_t *) PC) [21 ... 10] = (S+A) [63 ... 52]+`
+
+|71
+|R_LARCH_PCALA_HI20
+|32位相对PC偏移的[12:31]位
```

Comment:
This description seems different than the symbolic definition below.  Say PC is 0x0f00 and (S + A) is 0x1000, `(((S+A) & ~0xfff) - (PC & ~0xfff)) [31 ... 12]` will be 0x1000 but the PC-relative offset will be 0x0100, and "the [12:31] bits of PC-relative offset" will be zero.

Reply:
Added additional explanation: "all PC-relative offset calculations do not include the lowest 12 bits".